### PR TITLE
updated send(midiMessage&) to better handle system real time,

### PR DESCRIPTION
**Pull request by paul-emile-element**
sysex and other system common messages

### CODE REVIEW

File: src/midi_Message.h [hunk 1]
```diff
@@ -114,6 +114,18 @@ struct Message
         const unsigned size = unsigned(data2) << 8 | data1;
         return size > sSysExMaxSize ? sSysExMaxSize : size;
     }
+    inline bool isSysRT () const
```
Comment (franky47) [inline src/midi_Message.h:117]: Could we use fully qualified names (`isSystemRealTime` and `isSystemCommon`) for those methods please ?

**Review by franky47 — CHANGES_REQUESTED** [review]
LGTM, apart from the message method names.

I like the masking trick to tell apart channel/real time and common, great job!

File: src/midi_Message.h (hunk 1 above)
Comment (paul-emile-element) [reply src/midi_Message.h:117]: sure no problem.

**Review by franky47 — APPROVED** [review]

**Comment by franky47** [comment]
Thanks for this !

**Pull request merged**